### ISSUE TITLE
Docker build fails for kafka-nodejs-console-sample due to ubuntu:latest base image

### ISSUE THREAD

**Issue by osowski** [ibm-messaging/event-streams-samples#61]
The `kafka-nodejs-console-sample` docker build currently fails due to apt packages no longer being available on the `ubuntu:latest` container image. Since nothing else has changed in this folder of the repository in some time, I figured this was an issue with the underlying Ubuntu image versioning rather than code.

The output of the docker build as run today is below. You can see the issues with no candidates for `nodejs-dev` and `libssl1.0-dev` available. I have found that specifying the previous LTS version of Ubuntu 18.04 as the base image resolves this issue. I have built, tested, and verified the new container image based on Ubuntu 18.04 locally.

**Error:**
```
$ docker build -t eventstreams-samples-nodejs .
Sending build context to Docker daemon  58.37kB
Step 1/8 : FROM ubuntu
latest: Pulling from library/ubuntu
a4a2a29f9ba4: Pull complete
127c9761dcba: Pull complete
d13bf203e905: Pull complete
4039240d2e0b: Pull complete
Digest: sha256:35c4a2c15539c6c1e4e5fa4e554dac323ad0107d8eb5c582d6ff386b383b7dce
Status: Downloaded newer image for ubuntu:latest
 ---> 74435f89ab78
Step 2/8 : RUN  apt-get update -qqy   && apt-get install -y --no-install-recommends      build-essential      node-gyp      nodejs-dev      libssl1.0-dev      liblz4-dev      libpthread-stubs0-dev      libsasl2-dev      libsasl2-modules      make      python      nodejs npm ca-certificates   && rm -rf /var/cache/apt/* /var/lib/apt/lists/*
 ---> Running in f3379cc61eb1
Reading package lists...
Building dependency tree...
Reading state information...
Package libssl1.0-dev is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

Package nodejs-dev is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source
However the following packages replace it:
  libnode-dev

E: Package 'nodejs-dev' has no installation candidate
E: Package 'libssl1.0-dev' has no installation candidate
The command '/bin/sh -c apt-get update -qqy   && apt-get install -y --no-install-recommends      build-essential      node-gyp      nodejs-dev      libssl1.0-dev      liblz4-dev      libpthread-stubs0-dev      libsasl2-dev      libsasl2-modules      make      python      nodejs npm ca-certificates   && rm -rf /var/cache/apt/* /var/lib/apt/lists/*' returned a non-zero code: 100
```

**Fix:**
Change https://github.com/ibm-messaging/event-streams-samples/blob/master/kafka-nodejs-console-sample/Dockerfile#L8 to:
`FROM ubuntu:18.04`

**Comment by edoardocomar** [comment]
Thanks @osowski I had to clear my local docker cache to reproduce, as my builds were otherwise working

**Comment by mimaison** [comment]
Fixed in #62, closing
